kube/src/error.rs

```diff
@@ -1,5 +1,5 @@
 //! Error handling in [`kube`][crate]
```

Comment:
This file can be reverted since `error` is behind a feature now.

Reply:
i do think that `config` and` client` can be somewhat exclusive features (except that `client` would depend on `config`), so am going to leave this in the hopes that we can create a config only feature set.

Reply:
well, we can put it back in that case, actually, removing :-)